frontend/src/features/help/forms/BugForm.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { Formik } from 'formik';
+import { Form } from 'react-bootstrap';
+import { Input, TextArea } from 'components/common/form';
+import { noop } from 'lodash';
+import { pimsSupportEmail } from '../constants/HelpText';
+import { IHelpForm } from '../interfaces';
+
+interface BugFormProps {
+  /** Form values that should overwrite the default form values */
+  formValues: IHelpForm;
+  /** Call this function whenever the form fields are updated to keep the mailto in sync with this form */
+  setMailto: Function;
+}
+interface IBugForm extends IHelpForm {
+  stepsToReproduce: string;
+  expectedResult: string;
+  actualResult: string;
+}
+
+const defaultHelpFormValues: IBugForm = {
+  user: '',
+  email: '',
+  page: '',
+  stepsToReproduce: '',
+  expectedResult: '',
+  actualResult: '',
+};
+
+/**
+ * Form allowing user to report a bug. The state of this form is synchronized with the parent's mailto.
+ */
+const BugForm: React.FunctionComponent<BugFormProps> = ({ formValues, setMailto }) => {
+  const initialValues = { ...defaultHelpFormValues, ...formValues };
+  return (
+    <Formik
+      initialValues={initialValues}
+      onSubmit={noop}
+      validateOnMount={true}
+      validate={(values: any) => {
+        const body = `Steps to Reproduce: ${values.stepsToReproduce} Excepted Result: ${values.expectedResult} Actual Result: ${values.actualResult}`;
```

Comment:
I tried exceptionally hard to get the mailto to support newline characters, but I couldn't get gmail to work. GMail would either just print the newline characters as text or it would eat them and not display them at all.

tried \n \r\n \r %0D%0A <br> <br /> 

None of the above worked. Not sure what I'm missing.